src/mips64/cp0.rs

```diff
@@ -77,8 +77,26 @@ impl Cop for Cp0 {
             cop0: self,
         };
         match op.func() {
+            0x00 => {
+                // MFC0
+                match op.rd() {
+                    12 => {
+                        op.cpu.regs[op.rt()] = op.cop0.reg_status;
+                        op.cpu.tight_exit = true;
+                    }
+                    13 => {
+                        op.cpu.regs[op.rt()] = op.cop0.reg_cause;
+                        op.cpu.tight_exit = true;
+                    }
+                    _ => warn!(
+                        op.cop0.logger,
+                        "unimplemented COP0 read32";
+                        o!("reg" => op.rd())
```

Comment:
FWIW I think we don't need `o!` here, I didn't understand how the logging library worked at first. I'll eventually do a pass and remove it everywhere. You can leave it here if you want, it doesn't really matter.

Reply:
yep, works fine without it